web/src/pages/vulnerability/addvulnerability.tsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState } from "react";
```

Comment:
Please rename page name from `vulnerability/addvulnerability.tsx` to `Vulnerability/Add.tsx` so that the linter doesn't complain